handler/CMakeLists.txt

```diff
@@ -35,42 +34,68 @@ if(LINUX OR ANDROID)
 endif()
 
 if(LINUX)
-    target_sources(crashpad_handler PRIVATE
+    target_sources(crashpad_handler_lib PRIVATE
         linux/cros_crash_report_exception_handler.cc
         linux/cros_crash_report_exception_handler.h
     )
 endif()
 
 if(WIN32)
-    target_sources(crashpad_handler PRIVATE
+    target_sources(crashpad_handler_lib PRIVATE
         win/crash_report_exception_handler.cc
         win/crash_report_exception_handler.h
     )
 endif()
 
+target_link_libraries(crashpad_handler_lib
+    PRIVATE
+        $<BUILD_INTERFACE:crashpad_interface>
+    PUBLIC
+        crashpad_compat
+        crashpad_minidump
+        crashpad_snapshot
+        crashpad_util
+        mini_chromium
+)
+
+if(WIN32)
+    if(MSVC)
+        target_compile_options(crashpad_handler_lib PRIVATE "/wd4201")
+    endif()
+    if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+        target_compile_options(crashpad_handler_lib PRIVATE
+            "-Wno-multichar"
+        )
+    endif()
+endif()
+
+set_property(TARGET crashpad_handler_lib PROPERTY EXPORT_NAME handler)
```

Comment:
I considered doing the following to rename the final library from `crashpad_handler_lib.lib` to `crashpad_handler.lib`:

```
set_property(TARGET crashpad_handler_lib PROPERTY OUTPUT_NAME crashpad_handler
```

But I got build errors on Windows under some configurations because the executable and the library share the output directory and both have the same base name.